cpp/dolfinx/fem/utils.h

```diff
@@ -769,7 +770,8 @@ void pack_coefficients(const Form<T>& form, IntegralType integral_type, int id,
     {
     case IntegralType::cell:
     {
-      auto fetch_cell = [](auto& entity) { return entity.front(); };
+      auto fetch_cell
```

Comment:
Are the changes removing `auto` intended?

Reply:
It wasn't compiling. I think because `auto&` is wrong, but `auto` seems OK.

Reply:
I'm not sure it's wrong.

Have you checked performance? This is a critical part of the code - is the pass by value an issue?

Reply:
Did you test? It brings in a copy of the size at every call.